File: marti/Model/VintageModel/ILstepImage.m
//
//  ILstepImage.m
//  marti
//
//  Created by Paul Legault Local on 11-12-09.
//  Copyright (c) 2011 __MyCompanyName__. All rights reserved.
//

#import "ILstepImage.h"
#import "ILstep.h"


@implementation ILstepImage

@dynamic stepImage_data;
@dynamic stepImage_flags;
@dynamic stepImage_modifiedDate;
@dynamic stepImage_version;
@dynamic belongToStep;
@dynamic steppictogramid;


-(void)setPictogramid:(NSString*) pictId{
    self.steppictogramid = pictId;
}
@end

File: marti/Model/VintageModelImporter/ILstepImage.swift
//
//  ILstepImage.swift
//  
//
//  Created by Marco Guilmette on 2015-08-20.
//
//

import Foundation
import CoreData

class ILstepImage: NSManagedObject {

    @NSManaged var stepImage_data: NSData
    @NSManaged var stepImage_flags: String
    @NSManaged var stepImage_modifiedDate: NSDate
    @NSManaged var stepImage_version: NSNumber
    @NSManaged var belongToStep: ILstep

}
